packages/react-native-reanimated/android/build.gradle

```diff
@@ -334,25 +284,11 @@ dependencies {
     implementation "androidx.core:core:1.6.0"
 
     implementation "com.facebook.react:react-android" // version substituted by RNGP
-    if (JS_RUNTIME == "hermes") {
-        implementation "com.facebook.react:hermes-android" // version substituted by RNGP
-    }
```

Comment:
`react-native-reanimated` does not depend on Hermes, only `react-native-worklets` does.